### DISCUSSION TITLE
Why is python useful in the public health field

### DISCUSSION THREAD

**Discussion by Adriannaluger** [earthlab-education/Earth-Analytics-2023-01-Intro#103] [Awesome science]
After the first class discussion I found myself wondering how python or any coding for that matter could be useful for me in the future as I want to go into some public health field job. I began researching and found this article that outlines a lot of benefits of python in the health field. I found this really interesting because as I have never coded before I didnt know any benefits and uses for my future. This article really changed my mindset though and I found it very intersting. 

https://techvidvan.com/tutorials/python-in-healthcare/

**Comment by ellenalamont17** [comment]
This is an excellent in the cross-applicability of skillsets and unique implementations! Excellent work clarifying this for yourself. Hopefully it will give you inspiration for a project or future assignment where you can play with these ideas. My partner is a business analyst for a team of coders and data analysts working in the healthcare sector. They delve into a diverse range of topics in their various subteams from quantifying patient experience surveys, insurance systems management, patient care and health, disease/illness tracking and statistics, etc. I liked the example of robotics in the article. Skies the limit.
